framework/CHANGELOG.md

```diff
@@ -271,6 +272,8 @@ Yii Framework 2 Change Log
 - Bug #13128: Fixed incorrect position of {pos} string in ColumnSchemaBuilder `__toString` (df2)
 - Bug #13159: Fixed `destroy` method in `yii.captcha.js` which did not work as expected (arogachev)
 - Bug #13198: Fixed order of checks in `yii\validators\IpValidator` that sometimes caused wrong error message (silverfire)
+- Bug #13200: Creating Urls for routes specified in `yii\rest\UrlRule::$extraPatterns` did not work if no HTTP verb was specified (cebe)
```

Comment:
I don't think these should be here.

Reply:
It's not the first time when `git merge` messes up changelog :(